pygmt/helpers/utils.py

```diff
@@ -130,11 +130,16 @@ def data_kind(data=None, x=None, y=None, z=None, required_z=False):
     'file'
     >>> data_kind(data=xr.DataArray(np.random.rand(4, 3)))
     'grid'
+    >>> data_kind(data=xr.DataArray(np.random.rand(3, 4, 5)))
+    'image'
     """
     if isinstance(data, (str, pathlib.PurePath)):
         kind = "file"
     elif isinstance(data, xr.DataArray):
-        kind = "grid"
+        if len(data.dims) == 3:
+            kind = "image"
+        else:
+            kind = "grid"
```

Comment:
Much shorter version:
```suggestion
        kind = "image" if len(data.dims) == 3 else "grid"
```